documentation.md

```diff
@@ -3,12 +3,18 @@ layout: page
 title: Documentation
 ---
 
-The OSCAR project initially encompasses a number of different software projects.
-As the OSCAR system is still under development, there is no single source for user
-documentation. 
+Besides the
+<a href="https://oscar-system.github.io/Oscar.jl/dev/">documentation for the OSCAR system itself</a>,
+the following links to documentation for software projects encompassed by OSCAR
+are useful.
```

Comment:
This way the link to the Oscar docs still is a bit "hidden" as it just is a link in a text that refers to other manuals. How about something more like this:
```suggestion
You can find the OSCAR manual here:
 - [for the latest development version of Oscar.jl](https://oscar-system.github.io/Oscar.jl/dev)
 - [for the latest stable release of Oscar.jl](https://oscar-system.github.io/Oscar.jl/latest)

In addition, the following links to documentation for software projects
encompassed by OSCAR are useful.
```

Reply:
Good idea.

Reply:
Thanks, with changes just discussed verbally fine with me.